src/index.css

```diff
@@ -40,6 +40,8 @@
     --ring: 215 20.2% 65.1%;
 
     --radius: 0.5rem;
+    /* The MacOS titlebar takes up this space, other components should push below it */
+    --titlebar-height: 3.5rem;
```

Comment:
Consolidate size / padding; here the lightbox Dialog needs it to ensure it doesn't overlap with the titlebar.